packages/stakers/src/consensus.ts

```diff
@@ -58,6 +60,13 @@ export class Consensus extends StakerComponent {
       { dnpName: ConsensusClientHolesky.Nimbus, minVersion: "0.1.2" },
       { dnpName: ConsensusClientHolesky.Lodestar, minVersion: "0.1.3" }
     ],
+    [Network.Hoodie]: [
+      { dnpName: ConsensusClientHoodie.Lighthouse, minVersion: "0.1.2" },
```

Comment:
all hoodi packages versions should be 0.1.0 since will be published as "newRepo"